### PR TITLE
Unhide WFP ADAM maps

### PR DESCRIPTION
**Pull request by frozenhelium**
#### Seasonal Risk watch
    * Add popup in seasonal risk watch
    * Add Tooltip in sidebar
    * UI fixes and audit

### CODE REVIEW

File: src/root/views/RegionalRiskWatch/SeasonalRisk/CountryRiskBarChart/index.tsx [hunk 1]
```diff
@@ -54,30 +58,49 @@ function CountryRiskBarChart(props: Props) {
...
-              )}
+                    return (
+                      <React.Fragment key={hrd.hazard_type}>
+                        <div
+                          // data-tip
+                          // data-for={hrd.hazard_type}
+                          // Note: Tooltip is added for hover title
+                          title={`${hrd.hazard_type_display}: ${hrd.value}`}
+                          className={styles.hazardRiskBar}
+                          key={hrd.hazard_type}
```
Comment (tnagorra) [inline src/root/views/RegionalRiskWatch/SeasonalRisk/CountryRiskBarChart/index.tsx:80]: We don't need this key anymore

File: src/root/views/RegionalRiskWatch/SeasonalRisk/CountryRiskBarChart/index.tsx [hunk 2]
```diff
@@ -45,7 +44,12 @@ function CountryRiskBarChart(props: Props) {
         size="small"
         className={styles.heading}
       >
-        Countries by Risk
+        <div>COUNTRIES BY RISK</div>
+        <div className={styles.riskStatus}>
+          <div>Low</div>
+          <div>High</div>
+        </div>
+
```
Comment (tnagorra) [inline src/root/views/RegionalRiskWatch/SeasonalRisk/CountryRiskBarChart/index.tsx:52]:
```suggestion
```

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: src/root/views/RegionalRiskWatch/SeasonalRisk/CountryRiskBarChart/index.tsx (hunk 2 above)
Comment (puranban) [reply src/root/views/RegionalRiskWatch/SeasonalRisk/CountryRiskBarChart/index.tsx]: done

File: src/root/views/RegionalRiskWatch/SeasonalRisk/CountryRiskBarChart/index.tsx (hunk 1 above)
Comment (puranban) [reply src/root/views/RegionalRiskWatch/SeasonalRisk/CountryRiskBarChart/index.tsx]: done removed

**Review by tnagorra — APPROVED** [review]

**Review by puranban — APPROVED** [review]

**Pull request merged**